react/week4/user-search/src/userList.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useUser } from "./userContext";
+
+const UserList = () => {
+  const { users, loading, count, error } = useUser();
+  // const { values } = useUser()
+
+  return (
+    <ul style={{ listStyleType: "none" }}>
+      {loading ? (
```

Comment:
It is fine to use ternaries here, it works but it becomes hard to read when it starts to chain multiple ternary operations. Can there be a way to make it simpler to read?